### PR TITLE
feat(sdk/php): Add PHP-SDK docs

### PR DESCRIPTION
**Pull request by criskell**
*work in progress*

### CODE REVIEW

File: docs/sdk/php/what-is-php-sdk.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+id: what-is-php-sdk
+title: O que é o SDK de PHP?
+tags:
+  - api
+  - php
+---
+
+O SDK de PHP é um kit de ferramentas criado com o objetivo de integrar facilmente os serviços da Woovi e OpenPix em suas aplicações PHP.
+
+Para tanto, o SDK permite que utilize qualquer cliente HTTP compatível com [PSR-18](https://www.php-fig.org/psr/psr-18/), como [Guzzle](https://docs.guzzlephp.org/en/stable/) e [symfony/http-client](https://symfony.com/doc/current/http_client.html). Portanto, é possível utilizar qualquer plataforma de PHP, como Laravel e Symfony.
```
Comment (edumaciel10) [inline docs/sdk/php/what-is-php-sdk.md:11]: como Laravel, Symfony. Para integrações customizadas é possível utilizar também em ecommerces, como Magento, WooCommerce, PrestaShop etc

**Review by edumaciel10 — APPROVED** [review]
Great, I think that adding a link to repo will be great!

File: docs/sdk/php/usage.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
+---
+id: usage
+title: Como iniciar
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:3]: Como começar

File: docs/sdk/php/usage.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
+---
+id: usage
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:2]: sdk-php-usage

File: docs/sdk/php/usage.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+
+É necessário ter o PHP com versão superior à 7.4 instalado com o [Composer](https://getcomposer.org).
+
+A maneira mais fácil de instalar o SDK é com o comando abaixo que instala as dependências necessárias com o Composer:
+
+```bash
+$ composer require open-pix/php-sdk guzzlehttp/guzzle guzzlehttp/psr7
+```
+
+Dessa forma, o SDK, um cliente HTTP (`guzzlehttp/guzzle`) e uma implementação da [PSR-17](https://www.php-fig.org/psr/psr-17/) (`guzzlehttp/psr7`) serão instalados.
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:19]: what is the another way?

File: docs/sdk/php/usage.md (hunk 3 above)
Comment (criskell) [reply docs/sdk/php/usage.md:2]: sdk-php-getting-started?

File: docs/sdk/php/usage.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+
+## Dependências
+
+O SDK depende de implementações das PSRs 17 e [18](https://www.php-fig.org/psr/psr-18/).
+
+A PSR-17 é usada para fornecer interfaces para mensagens HTTP, tanto requisições quanto respostas. Por outro lado, a PSR-18 é usada para fornecer interfaces para clientes HTTP.
+
+Sendo interfaces, é necessário ter uma implementação instalada, como `guzzlehttp/guzzle` para a PSR-18 e `guzzlehttp/psr7` para a PSR-17.
+
+As PSRs são utilizadas para não depender diretamente de clientes HTTP, deixado a cargo do desenvolvedor.
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:29]: I recommend move it to the end

File: docs/sdk/php/usage.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+O ponto de entrada do SDK é um `Client`:
+
+```php
+use OpenPix\PhpSdk\Client;
+
+// Para utilizar OpenPix
+$client = Client::create("coloque seu appid aqui");
+```
+
+O método `create` cria um novo cliente a partir de um ID de aplicativo obtido no [site da OpenPix](https://app.wooopenpix.com.br/home/applications/tab/list).
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:42]: fix it link
[app.openpix.com.br/](https://app.openpix.com.br/home/applications/tab/list)

File: docs/sdk/php/usage.md [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+
+```php
+// Cria um cliente.
+$result = $client->customers()->create([
+    "name" => "Dan", // Nome
+    "taxID" => "31324227036", // TaxID
+    "email" => "email0@example.com", // E-mail
+    "phone" => "5511999999999", // Telefone
+    "correlationID" => "9134e286-6f71-427a-bf00-241681624586", // ID de correlação
+]);
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:62]: I would recommend put the customer data into an variable named as $dadosCliente

File: docs/sdk/php/usage.md [hunk 8]
```diff
@@ -0,0 +1,238 @@
...
+$client->customers();
+```
+
+Cada recurso irá ter um conjunto de métodos que podem serem executados para realizarem operações:
+
+```php
+// Cria um cliente.
+$result = $client->customers()->create([
+    "name" => "Dan", // Nome
+    "taxID" => "31324227036", // TaxID
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:58]: Notice to the customer that taxID is an CPF
"taxID" => "31324227036", // CPF

File: docs/sdk/php/usage.md [hunk 9]
```diff
@@ -0,0 +1,238 @@
...
+    "value" => 100,
+    "customer" => [
+        "name" => "Dan",
+        "taxID" => "31324227036",
+        "email" => "email0@example.com",
+        "phone" => "5511999999999",
+    ],
+    "dayGenerateCharge" => 15,
+]);
+```
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:90]: change the data array to an variable

File: docs/sdk/php/usage.md [hunk 10]
```diff
@@ -0,0 +1,238 @@
...
+
+// Retorna o nome do primeiro cliente da página 2.
+echo $page["customers"][0]["name"];
+```
+
+## PHPDocs
+
+Em cada operação disponível para um determinado tipo de recurso, existem PHPDocs disponíveis informando o formato de entrada e saída da operação com um link para a documentação da API Rest e exemplo de utilização.
+
+Para utilizar, é sugerido utilizar um editor com Intellisense como PhpStorm ou Visual Studio Code com a extensão Intelephense.
```
Comment (edumaciel10) [inline docs/sdk/php/usage.md:224]: give the link to the customer

**Review by edumaciel10 — APPROVED** [review]
cool!!

File: docs/sdk/php/usage.md (hunk 4 above)
Comment (criskell) [reply docs/sdk/php/usage.md:19]: installing another http client and another message factory. I will rephrase the sentence.

**Review by vinibgoulart — APPROVED** [review]

File: docs/sdk/php/what-is-php-sdk.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+---
+id: what-is-php-sdk
```
Comment (daniloab) [inline docs/sdk/php/what-is-php-sdk.md:2]: prefer `php-sdk-what-is` feature first

**Review by daniloab — APPROVED** [review]

**Review by daniloab — APPROVED** [review]

**Review by daniloab — APPROVED** [review]

**Review by daniloab — APPROVED** [review]

**Pull request merged**